src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -472,6 +484,16 @@ void AutomatedRssDownloader::handleRuleCheckStateChange(QListWidgetItem *ruleIte
     m_ui->listRules->setCurrentItem(ruleItem);
 }
 
+void AutomatedRssDownloader::clearDownloadedEpisodesSelectedRule()
+{
+    QMessageBox::StandardButton reply = QMessageBox::question(this, tr("Clear downloaded episodes"), tr("Are you sure you want to clear the downloaded episodes for the selected rule?"), QMessageBox::Yes | QMessageBox::No);
```

Comment:
The question is not strictly clear, as to me. I would add "list" to point out that there is a list of episodes and the list will be cleared, not episodes themselves.